src/sentry/api/serializers/models/integration.py

```diff
@@ -189,6 +189,7 @@ def get_attrs(self, item_list, user, *args, **kwargs):
             issues_by_integration[ei.integration_id].append({
                 'id': six.text_type(ei.id),
                 'key': ei.key,
+                'url': [i.get_installation(self.group.organization.id).get_issue_url(ei.key) for i in item_list if i.id == ei.integration_id][0],
```

Comment:
It would probably be better to use [`next()`](https://docs.python.org/2/library/functions.html#next) here with a generator expression ([see here](https://stackoverflow.com/questions/47789/generator-expressions-vs-list-comprehension)) so that we stop iteration once we find the first element.

```py
next(i.get_installation(self.group.organization.id).get_issue_url(ei.key) for i in item_list if i.id == ei.integration_id)
```